lib/LTP/utils.pm

```diff
@@ -129,16 +132,17 @@ sub log_versions {
 
     record_info('KERNEL VERSION', script_output('uname -a'));
     record_info('KERNEL DEFAULT PKG', script_output("cat $kernel_pkg_log", proceed_on_failure => 1));
-    record_info('KERNEL EXTRA PKG', script_output('rpm -qi kernel-default-extra', proceed_on_failure => 1));
+    record_info('KERNEL EXTRA PKG', script_output("rpm -qi kernel-default-extra", proceed_on_failure => 1));
 
-    record_info('KERNEL pkg', script_output('rpm -qa | grep kernel', proceed_on_failure => 1));
+    record_info('KERNEL pkg', script_output("rpm -qa | grep kernel", proceed_on_failure => 1));
```

Comment:
I guess the quotation mark changes could be reverted.

Reply:
Reverted.